src/docs/markdown/automatic-https.md

```diff
@@ -76,6 +76,8 @@ Automatic HTTPS never overrides explicit configuration.
 
 You can [customize or disable automatic HTTPS](/docs/json/apps/http/servers/automatic_https/) if necessary.
 
+<aside class="tip">Disabling redirects keeps the HTTPS port enabled as the default port implicitly assigned to an address. The HTTP port is only bound for redirects and for the ACME HTTP challenge. If you need to serve over HTTP, you must explicitly configure Caddy to do it. [See the examples](/docs/automatic-https#examples).</aside>
```

Comment:
There is mention of disabling it in that section, it's a section I viewed early on IIRC as it was a rather lengthy document and not all the information seemed important to understand what to do (or what I was doing wrong). It's been isolated as a tip to separate the context a little for the reason you bring up, that was meant to be less confusing actually..

It's about `disable_redirects` (which both Caddyfile and JSON support for config), so not about serving redirects, I could change to "serve content" unless there is a better way to phrase it(expose HTTP port, be reachable over the HTTP port, support connections over HTTP, etc).

I didn't have the bit about ACME HTTP challenge there, that was added as a suggestion to clarify that the HTTP port was implicitly redirecting to HTTPS in addition to being used for that challenge.

Examples section was linked to for more information on how to configure Caddy properly when using `disable_redirects`, so that you could access both HTTP and HTTPS, as this was not very straightforward.. I was originally under the belief that HTTP was the default implicit port, and that the `auto_https` feature was implicitly adding HTTPS support/connections, such that `disable_redirects` would implicitly still handle both ports, but do as the value states and just disable HTTP->HTTPS redirects.. while `off` would do that and not implicitly add HTTPS.